src/error-handler.js

```diff
@@ -0,0 +1,50 @@
+// jshint unused:false
+import path from 'path';
+import errors from './index';
+
+export default function(app) {
+  return function(error, req, res, next) {
+    if ( !(error instanceof errors.FeathersError) ) {
+      let oldError = error;
+      error = new errors.GeneralError(oldError.message, {errors: oldError.errors});
+
+      if (oldError.stack) {
+        error.stack = oldError.stack;
+      }
+    }
+
+    const code = !isNaN( parseInt(error.code, 10) ) ? parseInt(error.code, 10) : 500;
+
+    // Don't show stack trace if it is a 404 error
+    if (code === 404) {
+      error.stack = null;
+    }
+
+    res.status(code);
+
+    res.format({
+      'text/html': function() {
```

Comment:
We may want to take this one out (since it has the hardcoded filenames and assumes that `app.get('public')` is set.